legend-engine-executionPlan-execution-store-relational/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/result/RelationalResult.java

```diff
@@ -185,7 +195,11 @@ public RelationalResult(SQLExecutionResult sqlExecutionResult, RelationalInstant
 
     private void buildTransformersAndBuilder(ExecutionNode node, DatabaseConnection databaseConnection) throws SQLException
     {
-        boolean isDatabaseIdentifiersCaseSensitive = databaseConnection.accept(new DatabaseIdentifiersCaseSensitiveVisitor());
+        // TODO : epsstan : ask the data source about this ??
+        RelationalDatabaseConnection relationalDatabaseConnection = (RelationalDatabaseConnection) databaseConnection;
```

Comment:
(RelationalDatabaseConnection) databaseConnection -> this cast is problamatic, we can get legacy DatabaseConnection too